### PR TITLE
[SW-2652] Add Roadmap for Q4-2021/Q1-2022 to README.rst

### PR DESCRIPTION
**Pull request by mn-mikke**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -148,6 +148,25 @@ For more details regarding the internal or external backend, please see
...
+- [`SW-2646 <https://h2oai.atlassian.net/browse/SW-2646>`__] Add capability to ``H2OMOJOModel`` to calculate model metrics on arbitrary dataset
+- [`SW-2634 <https://h2oai.atlassian.net/browse/SW-2634>`__][`SW-2635 <https://h2oai.atlassian.net/browse/SW-2635>`__] Better interaction with a user during model training (report progress, show warnings from H2O backend)
+- Improvements of Sparkling Water conversion functions with the respect to training time of H2O algorithms
+- Add tutorials for the lesser known algorithms (``RuleFit``, ``GAM``, ...)
+
+Sparkling Water 3.38.0.1-1 (Summer 2022)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+- Add compatibility tests with different versions of AWS EMR
+- Redesign of K8s tests
+- Expose `UpliftTrees` and ``ExtendedIsolationForest`` in Sparkling Water
```
Comment (michalkurka) [inline README.rst:167]: Please move user-facing items to the top of the list

**Review by michalkurka — APPROVED** [review]
Thank you @mn-mikke

File: README.rst [hunk 2]
```diff
@@ -148,6 +148,25 @@ For more details regarding the internal or external backend, please see
...
 
+Roadmap
+-------
+
+Sparkling Water 3.36.0.1-1 (Winter 2021/2022)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+- [`SW-2629 <https://h2oai.atlassian.net/browse/SW-2629>`__] Rewrite ``H2OWord2Vec`` to be Sparkling Water feature estimator
+- [`SW-2633 <https://h2oai.atlassian.net/browse/SW-2633>`__] Add support for Apache Spark 3.2
+- [`SW-2646 <https://h2oai.atlassian.net/browse/SW-2646>`__] Add capability to ``H2OMOJOModel`` to calculate model metrics on arbitrary dataset
+- [`SW-2634 <https://h2oai.atlassian.net/browse/SW-2634>`__][`SW-2635 <https://h2oai.atlassian.net/browse/SW-2635>`__] Better interaction with a user during model training (report progress, show warnings from H2O backend)
```
Comment (krasinski) [inline README.rst:159]: minor thing but there's also the third ticket SW-2637

File: README.rst [hunk 3]
```diff
@@ -148,6 +148,25 @@ For more details regarding the internal or external backend, please see
...
+
+Sparkling Water 3.36.0.1-1 (Winter 2021/2022)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+- [`SW-2629 <https://h2oai.atlassian.net/browse/SW-2629>`__] Rewrite ``H2OWord2Vec`` to be Sparkling Water feature estimator
+- [`SW-2633 <https://h2oai.atlassian.net/browse/SW-2633>`__] Add support for Apache Spark 3.2
+- [`SW-2646 <https://h2oai.atlassian.net/browse/SW-2646>`__] Add capability to ``H2OMOJOModel`` to calculate model metrics on arbitrary dataset
+- [`SW-2634 <https://h2oai.atlassian.net/browse/SW-2634>`__][`SW-2635 <https://h2oai.atlassian.net/browse/SW-2635>`__][`SW-2637 <https://h2oai.atlassian.net/browse/SW-2637>`__] Better interaction with a user during model training (report progress, show warnings from H2O backend)
+- Improvements of Sparkling Water conversion functions with the respect to training time of H2O algorithms
+- Add tutorials for the lesser known algorithms (``RuleFit``, ``GAM``, ...)
+
```
Comment (neemah2o) [inline README.rst:162]: Early 2022, we can add:
```
- Add IBM Spectrum Conductor SW launch scripts to repo and add integration tests (using IBM SC docker images)
```

**Pull request merged**